common-pipeline/pipeline-flow/pipeline-flow-v3-schema.json

```diff
@@ -200,6 +208,10 @@
           "description": "Unique identifier for the supernode within the current pipeline",
           "type": "string"
         },
+        "description": {
+          "description": "Super node description",
```

Comment:
We generally write it as 'Supernode' -- all one word.

Reply:
addressed. Thanks